examples/transactions/examples/transfer_sign.rs

```diff
@@ -0,0 +1,42 @@
+use alloy::network::{eip2718::Encodable2718, TransactionBuilder};
+use alloy::primitives::utils::parse_units;
+use alloy::providers::Provider;
+use alloy::{
+    network::EthereumSigner, node_bindings::Anvil, primitives::U256, providers::ProviderBuilder,
+    rpc::types::eth::TransactionRequest, signers::wallet::LocalWallet,
+};
+use eyre::Result;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let anvil = Anvil::new().try_spawn()?;
+
+    // Set up signer from the first default Anvil account (Alice).
+    let signer: LocalWallet = anvil.keys()[0].clone().into();
+    let destination: LocalWallet = anvil.keys()[1].clone().into();
```

Comment:
No need to cast to a local wallet for the destination

Please use the `alice` and `bob` nomenclature

https://github.com/alloy-rs/examples/blob/096abf02af7d771aaeb88f9d98d3a46727dc74e6/examples/transactions/examples/transfer_erc20.rs#L24-L26